src/master/develop/reading.md

```diff
@@ -351,6 +382,11 @@ This feature is not yet supported or not documented.
 Do you want to make a feature request or help out? Open an issue on [Java sdk github project](https://github.com/codenotary/immudb4j/issues/new)
 :::
 
+::: tab .NET
+This feature is not yet supported or not documented.
```

Comment:
I think I missed that in SDK - just asking for confirmation that we don't support GetAtRevision there.

Reply:
It is there. Adding the sample now.